features/step_definitions/deposit_steps.rb

```diff
@@ -4,36 +4,24 @@
 require 'json'
 
 Given("a preserved Bentley audio artifact") do
-  @artifact ||= Fabricate(:stored_package, content_type: "audio")
-  Fabricate(:queue_item, package: @artifact)
+  artifact = Fabricate(:stored_package, content_type: "audio")
+  Fabricate(:queue_item, package: artifact)
+
+  @artifact_bag_id = artifact.bag_id
+  @artifact_package_id = artifact.id
 end
 
 Given("I am a Bentley audio content steward") do
-  @key ||= Keycard::DigestKey.new
-  @user ||= Fabricate(:user, api_key_digest: @key.digest)
-  Services.checkpoint.grant!(
-    @user,
-    Checkpoint::Credential::Role.new('content_manager'),
-    Checkpoint::Resource::AllOfType.new('audio')
-  )
+  make_me_a("content_manager").on_all("audio")
 end
 
 When("I check the status of the artifact") do
-  header("Authorization", "Token token=#{@key}")
-  begin
-    @bags_response ||= get("/v1/bags/#{@artifact.bag_id}")
-  rescue ActiveRecord::RecordNotFound
-    @bags_response ||= Rack::MockResponse.new(404, {}, '')
-  end
+  @bags_response = api_get("/v1/bags/#{@artifact_bag_id}")
 
-  begin
-    queue_response_body ||= JSON.parse(get("/v1/queue?package=#{@artifact.id}").body)
-
-    @latest_queue_response_body ||= queue_response_body.reduce do |lhs, rhs|
-      DateTime.parse(lhs["updated_at"]) > DateTime.parse(rhs["updated_at"]) ? lhs : rhs
-    end
-  rescue JSON::ParserError
-    @latest_queue_response_body ||= {}
+  queue_response_body = json_get("/v1/queue?package=#{@artifact_package_id}",
+                                 default_on_failure: [])
+  @latest_queue_response_body = queue_response_body.reduce do |lhs, rhs|
+    DateTime.parse(lhs["updated_at"]) > DateTime.parse(rhs["updated_at"]) ? lhs : rhs
```

Comment:
I don't think the queue item is relevant to the feature as implemented. The artifact (package) exposes its status and that is sufficient.

When we have deposits and artifacts with clear and separate responsibilities, this second half could be related to the deposit. I would just comment this out for now.

Reply:
We talked about this on slack; the queue item is in fact relevant.